doc.go

```diff
@@ -797,28 +797,17 @@ and before retrieving the results.
 		...
 
 		func DemonstrateAsyncMode(db *sql.DB) {
-			// Enable asynchronous mode.
-			ctx := WithAsyncMode(context.Background())
-			// Establish connection
-			conn, _ := db.Conn(ctx)
-
-			// Unwrap connection
-			err = conn.Raw(func(x interface{}) error {
-				// Execute asynchronous query
-				rows, _ := x.(driver.QueryerContext).QueryContext(ctx, "select 1", nil)
-				defer rows.Close()
-
-				// Retrieve and check results of the query after casting the result
-				status := rows.(SnowflakeResult).GetStatus()
-				if status == QueryStatusComplete {
-					// do something
-				} else if status == QueryStatusInProgress {
-					// do something
-				} else if status == QueryFailed {
-					// do something
-				}
-				return nil
-			})
+			// Enable asynchronous mode
+			ctx := sf.WithAsyncMode(context.Background())
+
+			// Run the query with asynchronous context
+			rows, _ := db.QueryContext(ctx, "select 1")
```

Comment:
is it a good practice to ignore the second parameter? is it an error? I understand that if the passed query is wrong then we receive quick answer that sth is wrong?

Reply:
It is a common practice to skip the errors - in this case I wanted to keep the example, however you are right. This has been addressed.